docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,154 @@
+
+## Verify automatic migration
+
+List the existing Api objects after the Kyma upgrade is finished. Run this command:
+
+```shell script
+kubectl get apis --all-namespaces
+```
+
+As Api objects are removed after a successful migration, all the listed resources should be considered not migrated. Fetch logs from the job's pod to learn the reason for skipping the migration. 
+
+```shell script
+kubectl logs api-gateway-api-migrator-job -n kyma-system
+```
+
+
+If the Api was not migrated due to the invalid status or a blacklisted label, you either shouldn't or will not be able to do the migration.
+
+## Manual migration
+
+The manual migration process consists of the following steps:
+
+1. Fetch the Api resource you want to migrate:
+
+```shell script
+kubectl get api <NAME> -n <NAMESPACE> -o yaml
+```
+
+2. Create an APIRule resource based on the original Api object
+
+Here is the documentation for both custom resources, which may be useful during the migration:
+
+https://kyma-project.io/docs/1.11/components/api-gateway/#custom-resource-api-sample-custom-resource
+https://kyma-project.io/docs/1.11/components/api-gateway-v2#custom-resource-api-rule
+
+Only the `spec` part of the CRD will be described. `metadata` field can be adjusted in any way and `status` field should not be copied.
+
+Starting from the `service` field of APIRule:
+ * Copy `name` and `port` fields from the `service` field of Api object. 
+ * Set `host` value to any temporary value including the domain. For example, if the `hostname` value of Api was set to `sample-service.kyma.local` you can change it to `temp-sample-service.kyma.local`. Just make sure that the hostname is not used by other services on your cluster. That value will be changed in a later step, but for now it needs to be different than the original value. 
+ * Set `gateway` field of APIRule to the Istio Gateway used to expose services. By default it will be `kyma-gateway.kyma-system.svc.cluster.local`.
+
+The configuration of the `rules` field of APIRule is more complex and depends on the `authentication` configuration of APIRule. As all the basic scenarios are covered by automatic migration, this explanation will only concern the configurations that are not handled automatically.
+
+The basic difference between Apis and APIRules authentication configuration is that while Api allows to enable the authentication for the whole service and disable it on specific paths only, the APIRule has an approach where you specify what authentication should be used per specific path (including the possibility to set it for all paths) and the paths must not overlap. Another important difference is that Api supports a list of issuers and jwks URIs, but excluded paths are set independently on both. In the example below:
```

Comment:
```suggestion
The most important difference between Apis and APIRules authentication configuration is that Api is configured with a list of JWT issuers and APIRule is configured with a list of path definitions.
In the Api you can define a list of excluded paths for an issuer. Requests for these paths will not require authentication by this issuer but will require authentication from other configured issuers.
The APIRule has an approach where you specify what authentication should be used per the path definition.
Because paths are defined using regular expressions, you can define an expression that matches every path, or a subset of possible paths, or a single one.
Remember that path definitions in the APIRule must not overlap - if they do, requests to such paths are rejected. That's why translating an Api with multiple issuers having different excluded paths requires sophisticated regex definitions in a corresponding APIRule.
In the example below:
```

Reply:
done